packages/eas-cli/src/config.ts

```diff
@@ -0,0 +1 @@
+export const PAGE_LIMIT = 10000; // ten thousand
```

Comment:
Is this the right filename/place for this?

Reply:
I'd just hard-code it in places that need this for now, easy to extract later. This also avoids the problem of `PAGE_LIMIT` being maybe too generic as a name (what pages does this apply to?).

```suggestion
export const PAGE_LIMIT = 10_000;
```

Reply:
oh this is a cool trick